uniprot-search/src/main/java/org/uniprot/store/search/domain2/UniProtSearchFields.java

```diff
@@ -1,42 +1,65 @@
 package org.uniprot.store.search.domain2;
 
-import org.uniprot.core.cv.xdb.UniProtXDbTypes;
-import org.uniprot.store.search.domain2.impl.SearchFieldImpl;
-
 import java.util.List;
+import java.util.Objects;
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import org.uniprot.core.cv.xdb.UniProtXDbTypes;
+import org.uniprot.store.search.domain2.impl.SearchFieldImpl;
+
 /**
- * Represents all accessible UniProtKB search fields, and provides access to them via the {@link
- * SearchFields} contract.
+ * Represents all accessible search fields for UniProt domains (e.g., UniProtKB, UniParc, UniRef),
+ * and provides access to them via the {@link SearchFields} contract.
  *
- * <p>Created 12/11/2019
+ * <p>Created 17/01/2020
  *
  * @author Edd
  */
-public enum UniProtKBSearchFields implements SearchFields {
-    INSTANCE;
+public enum UniProtSearchFields implements SearchFields {
```

Comment:
The class name doesn't capture the sort part

Reply:
Sorting is a property of a search field.